docs/configuration-v3_0.md

```diff
@@ -2,7 +2,7 @@
 layout: default
 title: Config Spec v3.0.0
 parent: Configuration specifications
-nav_order: 3
+nav_order: 4
```

Comment:
Let's change these to count down from 50 the way the Butane docs do, so we don't have to renumber every spec every time.